src/systems/screenRenderer.js

```diff
@@ -1,47 +1,74 @@
-import { System } from 'ecsy';
-import { Phaser,Tile } from '../components';
+import { System } from "ecsy";
+import { Enemy, Location, Path, Phaser, Tile, Sprite } from "../components";
 
 export default class GridRenderSystem extends System {
   execute() {
     const game = this.queries.phaser.results[0].getComponent(Phaser).game;
-    const activeScene = game.scene.scenes.filter(s => game.scene.isVisible(s.scene.key))[0];
+    const activeScene = game.scene.scenes.filter((s) =>
+      game.scene.isVisible(s.scene.key)
+    )[0];
 
     // Add things to the active scene outside the switch to add to all scenes.
     activeScene.add.image(500, 500, "brownRock");
 
     // Add UI elements to particular scenes
-    activeScene.add.text(700,700,activeScene.scene.key);
-    switch (activeScene.scene.key){
+    activeScene.add.text(700, 700, activeScene.scene.key);
+    switch (activeScene.scene.key) {
       case "menu":
-        //activeScene.add.text(700,700,"menu")
         break;
       case "prep":
-        //activeScene.add.text(700,700,"prep")
-        //  break;
+      //  break;
       case "play":
+        const activeEnemies = this.queries.enemies.results;
+        activeEnemies.forEach((ent) => {
+          const enemyLocation = ent.getComponent(Location);
+          const xLocation = enemyLocation.x * 100 + 100;
+          const yLocation = enemyLocation.y * 20 + 100;
```

Comment:
These magic numbers are no good. We should probably have some utility that does the grid to screen space conversion. It doesn't need to be a `System`, just some util we can import places so we don't do different math in different places.

Also, the enemy component already has a speed attribute that should be factored into the formula for movement once you make things move smoothly from point to point.

Also also, that movement should happen in the enemy systems, not here even though you have the enemies here.

Reply:
Ill write the util right now. I think changing the other stuff is going to wait for the next gjpr.

Reply:
TODO make util